crates/cranelift/src/wasmfx/optimized.rs

```diff
@@ -322,6 +322,14 @@ pub(crate) mod typed_continuation_helpers {
         pointer_type: ir::Type,
     }
 
+    /// Compile-time representation of `crate::runtime::vm::fibre::FiberStack`.
+    pub struct FiberStack {
+        /// This is NOT the "top of stack" address of the stack itself. In line
+        /// with how the (runtime) `FiberStack` type works, this is a pointer to
+        /// the TOS address.
+        pointer_to_tos: ir::Value,
```

Comment:
How about
```suggestion
        tos_addr: ir::Value,
```
?

Reply:
I'm not in love with the curret name, but the "top of stack" value that this field point to is itself an address, so `tos_addr` doesn't quite communicate the slightly awkard fact that you need to dereference this field in order to get the top of stack...

Reply:
When I see `_addr` I expect to have to do some sort of lookup. Maybe simply `tos_ptr`? will do?

Reply:
Okay, I went with `tos_ptr`